core/flyout_vertical.js

```diff
@@ -343,7 +343,7 @@ Blockly.VerticalFlyout.prototype.wheel_ = function(e) {
       delta *= 10;
     }
     var metrics = this.getMetrics_();
-    var pos = metrics.viewTop + delta;
+    var pos = -this.workspace_.scrollY + delta;
```

Comment:
Can you explain why you went from a positive value (metrics.viewTop) to a negative value?